scRNA-seq/post-processing_scRNA-seq.Rmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "scRNA-seq_post_processing"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2019**
+
+In this notebook, we'll perform quality control analyses and normalization of 
+scRNA-seq count data. 
+
+As opposed to bulk rna-seq, there are there are a few main things to look out for
+in single cell RNA-seq:
+
+**Single-cell rna-seq has...**  
+- More amplification (and more of it's associated biases and error)  
+- More 0's in the gene expression data (most genes aren't expressed across cell types)  
+
+This script is adapted from 
+[Seurat pbmc tutorial](https://satijalab.org/seurat/pbmc3k_tutorial.html)
+  
+#### Set Up 
+For these analyses, we will need Seurat a lot, so we will go ahead and attach it.  
+```{r Set Up}
+# Attach library
+library(Seurat)
+```
+
+#### Import single cell RNA-seq counts matrix
+Here we are importing a counts matrix from single cell rna-seq data that has 
+already been processed using Salmon and tximport and saved the counts to a tsv 
+file. 
+This [data set](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84465) 
+is glioblastoma data that was Fluorescent Activated Cell sorted 
+and then was processed by paired end sequencing using smart-seq2 protocol 
+[(Darmanis et al, 2017).](https://www.ncbi.nlm.nih.gov/pubmed/29091775) 
```

Comment:
Here's the citation style we use elsewhere: Harenza, et al. _Scientific Data._ 2017.